core/src/jvmMain/kotlin/BuidConfig.kt

```diff
@@ -0,0 +1,6 @@
+public actual object BuildConfig {
+    public actual val isDebug: Boolean
+        // TODO: need to determine a good way to set this on JVM presuming we don't want to bundle BuildKonfig in the
+        //  library.
+        get() = true
```

Comment:
Didn't want to add the dependency on `BuildKonfig` to the library, will explore if there is an alternative.